easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
@rjeschmi: support for overriding the `name` bit of the module name is implemented in https://github.com/hpcugent/easybuild-framework/pull/1389 (still WIP)